tests/request.rs

```diff
@@ -0,0 +1,92 @@
+use actix_web::{
+    test::{self, read_body, read_body_json, TestRequest},
+    web, App,
+};
+use std::{collections::BTreeMap, fs, path::PathBuf, str::from_utf8};
+
+use sol_to_uml::{
+    sol_to_uml_handler,
+    types::{SolToUmlRequest, SolToUmlResponse},
+};
+
+const CONTRACTS_DIR: &'static str = "tests/contracts";
+const UML_DIR: &'static str = "tests/uml";
+const ROUTE: &'static str = "/sol2uml";
+
+mod success_tests {
+    use super::*;
+
+    #[actix_web::test]
+    async fn contract_with_lib() {
+        let app = test::init_service(
+            App::new().service(web::resource("/sol2uml").route(web::post().to(sol_to_uml_handler))),
+        )
+        .await;
+
+        let contract_path = format!("{}/contract_with_lib.sol", CONTRACTS_DIR);
+        let uml_path = format!("{}/contract_with_lib.svg", UML_DIR);
+        let contract =
+            fs::read_to_string(&contract_path).expect("Error while reading contract_with_lib.sol");
+        let uml = fs::read_to_string(&uml_path).expect("Error while reading contract_with_lib.svg");
+
+        let request = SolToUmlRequest {
+            sources: BTreeMap::from([(PathBuf::from("contract_with_lib.sol"), contract)]),
+        };
+        let response = TestRequest::post()
```

Comment:
it's okay to test via http requests, but usually people do this to test the logic of the json converting, server starting, proper http status codes in response and etc. When you just want to test the logic of your function you can just... call your function :) 
You can leave this test, it's good, but if you will add more test files, I suggest you to just call the function directly